libcache/cache.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Graphics library
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Malgorzata Wrobel
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include "cache.h"
+
+#include <stdlib.h>
+#include <string.h>
+
+cacheline_t cache_createLine(uint64_t tag, uint32_t *data)
+{
+	cacheline_t line = { tag, data, '1' };
+
+	return line;
+}
+
+cacheset_t *cache_createSet(void)
+{
+	cacheset_t *cacheSet = malloc(sizeof(cacheset_t));
+
+	if (cacheSet == NULL) {
+		return NULL;
+	}
+
+	cacheSet->timestamps = calloc(NUM_WAYS, sizeof(cacheline_t *));
+	cacheSet->tags = calloc(NUM_WAYS, sizeof(cacheline_t *));
+	cacheSet->lines = calloc(NUM_WAYS, sizeof(cacheline_t));
```

Comment:
I think lines were supposed to be a static table within `cacheset_t`

Reply:
Changes have been made. Thank you